.github/workflows/push.yml

```diff
@@ -1,15 +1,24 @@
 on: push
```

Comment:
Housekeeping as well, we don't update this repository often so doing similar changes that we did on the admin/API